python/ray/tests/conftest.py

```diff
@@ -52,6 +52,8 @@ def get_default_fixture_ray_kwargs():
 def _ray_start(**kwargs):
     init_kwargs = get_default_fixture_ray_kwargs()
     init_kwargs.update(kwargs)
+    if os.environ.get("RAY_bootstrap_with_gcs") in ("true", "1"):
+        init_kwargs["gcs_server_port"] = 9993
```

Comment:
Any reason not to use the (previous) redis port?

Reply:
redis port by default is that port. so I'm thinking maybe gcs shouldn't use that. but for test i can update it to that.